### PR TITLE
Person Mapping

### PR DESCRIPTION
**Pull request by MehreenT**
Mapping of person entity.
Added rspec test cases for person entity.

### CODE REVIEW

File: app/models/entities/person.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    output[:lead_referent_type] = 'Entity::Person'
+
+    output
+  end
+
+  map from('first_name'), to('first_name')
+  map from('last_name'), to('last_name')
+  map from('email/address'), to('email[0]/value')
+  map from('organization_id'), to('company_id')
+  map from('lead_referent_id'), to('owner_id/id')
```
Comment (MarcoBgn) [inline app/models/entities/person.rb]: If you are referring to a User this should be `assignee_id`

File: app/models/entities/person.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  # Mapping to PipeDrive
+  after_normalize do |input, output|
+    output[:name] = [input['first_name'], input['last_name']].join(' ').strip
+
+    output
+  end
+
+  # Mapping to Connec!
+  after_denormalize do |input, output|
+    output[:is_customer] = true
```
Comment (MarcoBgn) [inline app/models/entities/person.rb]: You can omit this as is the default

File: app/models/entities/person.rb (hunk 1 above)
Comment (x4d3) [inline app/models/entities/person.rb]: maybe you can map phone ?
```
  "phone": [
            {
                "label": "",
                "value": "012345967",
                "primary": true
            }
        ],
```

File: app/models/entities/person.rb (hunk 2 above)
Comment (MehreenT) [reply app/models/entities/person.rb]: Feedback incorporated.

File: app/models/entities/person.rb (hunk 1 above)
Comment (MehreenT) [reply app/models/entities/person.rb]: Thank you for the feedback, mapping is updated as per your suggestion.

Comment (MehreenT) [reply app/models/entities/person.rb]: Thank you for the feedback, the mapping of phone is added.

**Pull request merged**